README.md

```diff
@@ -84,7 +84,7 @@ Note that although babel supports ES2015 `import` statements, [the `babel-node`
 
 ```
 npm install creed
-npm install -g babel-node
+npm install -g babel-node-cli
```

Comment:
Thanks @dmitriz!  I couldn't find a `babel-node-cli` package on npm.  Did you mean `babel-cli`?  If so, I'll be happy to merge after a quick fix to the name.